src/components/List/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Category from '../../interfaces/Category'
+import Product from '../../interfaces/Product'
+
+interface ListProps {
+  items: Array<Category|Product>,
+  itemComponent: any
+  prop: string;
+}
+
+export default function List ({
```

Comment:
This component doesn't seem to do much. Basically, it's only doing the `.map` of the products or categories, so my comment is remove it.

Reply:
I was applying a List component behavior. In my previous **Deliverable** I got a comment that I was applying the same code, then I could abstract it into a Component